### DISCUSSION TITLE
Difference beterrn TRUE and true in README.md - Does it matter?

### DISCUSSION THREAD

**Discussion by Subnet-Masked** [itzg/docker-minecraft-server#1853] [Q&A] [answered]
I am reading through the README.md trying to configure a new server - For some context I am very new to Kubernetes/Docker.

I noticed that some environment variables use "true" and "false" (lowercase) and some user "TRUE" and "FALSE" (uppercase). What I am asking is if they are interchangeable or if I need to make sure to use all upper or all lower for respective environment variables?

Thanks

**Comment by itzg** [comment, marked as answer]
Sorry for the inconsistencies in the docs. They are processed case-insensitive.

**Reply by Subnet-Masked** [reply]
Thank you for clarifying. Happy crafting!
